spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/encryption/EncryptionController.java

```diff
@@ -156,7 +157,7 @@ public String decrypt(@PathVariable String name, @PathVariable String profiles,
 					profiles, input);
 			TextEncryptor encryptor = this.encryptor.locate(encryptorKeys);
 			String encryptedText = this.helper.stripPrefix(input);
-			String decrypted = this.helper.stripPrefix(encryptor.decrypt(encryptedText));
+			String decrypted = encryptor.decrypt(encryptedText);
```

Comment:
Removing extraneous `stripPrefix()` which was masking the bug in `encrypt()`. While this is likely correct long term, you may consider leaving it in for a deprecation period because some users using `EncryptionController` for round-trip encryption may have stored values with the prefix included.